src/mixins/raster-layer-windbarb-mixin.js

```diff
@@ -4241,8 +4241,8 @@ export default function rasterLayerWindBarbMixin(_layer) {
   //     chart.pushLayer("pointmap", _layer);
   //   }
   // };
-  // _layer.getLayerNames = function(chart) {
-  //   return chart.getLayerNames();
-  // };
+  _layer.getLayerNames = function(chart) {
```

Comment:
On the surface it feels like the caller would need to call this method like so:

```
layer.getLayerNames(chart)
```
but that seems superfluous. Why not just call the `getLayerNames()` method directly from `chart` in that case: 

```
chart.getLayerNames()
```